Comparison Engine/Comparison Engine/Form1.cs

```diff
@@ -136,7 +137,13 @@ private void openChildFormDrink(Drink drink)
             //closeActiveForm();            //#commentedarea
             configureChildForm(drinkForm);
         }
-        private void configureChildForm(Form childForm)
+        private void openChildFormUserContribution(Drink drink, Bar bar)
+        {
+            Child_Forms.UserContribution userContribution = new UserContribution(drink, bar);
```

Comment:
"Child_Forms." shouldn't be needed, since you already have the namespace imported